### PR TITLE
REFPLTB-559 Switch from musl to glibc

### PR DESCRIPTION
**Pull request by scthunderbolt**
glibc is needed to build rbus and rbus-core.

### CODE REVIEW

File: conf/machine/turris-extender.conf [hunk 1]
```diff
@@ -15,7 +15,7 @@ MACHINE_EXTRA_RRECOMMENDS += "kernel-modules"
 IMAGE_BOOT_FILES = "zImage zImage-armada-385-turris-omnia.dtb;armada-385-turris-omnia.dtb boot.scr"
 
 MACHINE_IMAGE_NAME = "rdkb-generic-extender-image"
-TCLIBC = "musl"
```
Comment (rajkumar154) [inline conf/machine/turris-extender.conf]: Khem suggested to use musl library only in place of glibc. Usage of glibc is forbidden for TurrisOmnia target.

Comment (scthunderbolt) [reply conf/machine/turris-extender.conf]: Then we would have to suspend Turris Omnia build, since rbus does not build with musl.

**Comment by scthunderbolt** [comment]
The build error is:
"
| /home/jenkins/jenkinsroot/workspace/build-tur-bb/EXEC_0/build-turris/tmp/work/armv7ahf-neon-rdk-linux-musleabi/rbus-core/1.99-r0/git/rbus-core/lib/rbus_core.cpp:224:34: error: 'PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP' was not declared in this scope
|  static pthread_mutex_t g_mutex = PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP;
|                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
"

**Comment by scthunderbolt** [comment]
It has been decided on a call with Khem (20 Mar) that the broadband profile can switch to glib temporary and leave the pod profile with musl.

When the rbus / rbus-core problem is fixed with musl, then the bb profile should switch back to musl from glib.

A new PR will be submitted.

**Pull request closed without merging**